doc/classes/Node.xml

```diff
@@ -464,6 +464,12 @@
 				Fetches a node by [NodePath]. Similar to [method get_node], but does not generate an error if [param path] does not point to a valid node.
 			</description>
 		</method>
+		<method name="get_node_rpc_config" qualifiers="const">
+			<return type="Variant" />
+			<description>
+				Returns a [Dictionary] mapping method names to their RPC configuration defined for this [Node] using [method rpc_config].
```

Comment:
We do not need to refer to the class here, it's a bit more cumbersome to read and localize.
```suggestion
				Returns a [Dictionary] mapping method names to their RPC configuration defined for this node using [method rpc_config].
```